### ISSUE TITLE
Botania r183 is assigning empty NBT to unconverted items dropped into manapools

### ISSUE THREAD

**Issue by Barhandar** [Vazkii/Botania#815]
Tags are empty. Other possible culprit is addition of Storage Drawers, no other changes were made since it was generating non-tagged redstone.

**Comment by Barhandar** [comment]
Additionally, I'm utilizing "redropping", aka taking redstone that was just picked up and dropping it back into the pool for faster conjuring. Which seems to glitch, as, for example, I've just dropped 27 tagged redstone in, and got 54 normal redstone back... plus 7 redstone, tagged, on an almost run-out pool.

Trying to convert with not enough mana is practically guaranteed to return tagged item. E.g. I've dropped 64 redstone in a pool with enough mana to convert 13, and got 26 normal redstone back and 51 tagged redstone.
